.github/main.workflow

```diff
@@ -0,0 +1,9 @@
+workflow "API extractor" {
+  resolves = ["Breaking change detection bot"]
+  on = "pull_request"
+}
+
+action "Breaking change detection bot" {
+  uses = "./.github/api-extractor-action"
+  secrets = ["GITHUB_TOKEN"]
+}
```

Comment:
The whole api diff idea is pretty cool.  I may be getting greedy, but is it possible to see from which class/source file the functions belong to in the diff report?

<img width="782" alt="Screen Shot 2019-07-16 at 2 04 08 PM" src="https://user-images.githubusercontent.com/7065346/61318755-c8c80300-a7d3-11e9-9bc2-919482ef19e7.png">

Reply:
That would require using some diff algorithm that would be typescript aware and would keep important lines (in terms of context) in diff.
We can think about it later if this bot would overall be helpful.